gammapy/irf/effective_area_table.py

```diff
@@ -542,6 +545,8 @@ def evaluate(self, offset=None, energy=None):
             val = self._eval_spline(offset.value, np.log10(energy.value))
         else:
             raise ValueError('Invalid interpolation method: {}'.format(method))
+
+        val = np.reshape(val, np.append(offset_shape, energy_shape))
```

Comment:
Are you sure this is necessary? In principle the reshaping should happen in line 572. Did you have problems with the output shape?